tests/lib/rules/no-array-prototype-extensions.js

```diff
@@ -408,10 +408,56 @@ set.filter(item => get(item, "age") === 18);`,
       errors: [{ messageId: 'main', type: 'CallExpression' }],
     },
     {
+      // When unexpected number of arguments are passed, auto-fixer doesn't get triggered
       code: 'something.findBy()',
       output: null,
       errors: [{ messageId: 'main', type: 'CallExpression' }],
     },
+    {
+      // When unexpected number of arguments are passed, auto-fixer doesn't get triggered
+      code: 'something.findBy(1, 2, 3)',
+      output: null,
+      errors: [{ messageId: 'main', type: 'CallExpression' }],
+    },
+    {
+      // findBy with single argument
+      code: "something.findBy('abc')",
+      output: `import { get } from '@ember/object';
+something.find(item => get(item, 'abc'))`,
+      errors: [{ messageId: 'main', type: 'CallExpression' }],
+    },
+    {
+      // findBy with two arguments
+      code: "something.findBy('abc', 'def')",
+      output: `import { get } from '@ember/object';
+something.find(item => get(item, 'abc') === 'def')`,
+      errors: [{ messageId: 'main', type: 'CallExpression' }],
+    },
+    {
+      // findBy with two arguments and @ember/object's `get` is already imported
+      code: `import { get } from '@ember/object';
+      something.findBy('abc', 'def')`,
+      output: `import { get } from '@ember/object';
+      something.find(item => get(item, 'abc') === 'def')`,
+      errors: [{ messageId: 'main', type: 'CallExpression' }],
+    },
+    {
+      // findBy with two arguments and @ember/object's `get` is imported with an alias
+      code: `import { get as g } from '@ember/object';
+      something.findBy('abc', 'def')`,
+      output: `import { get as g } from '@ember/object';
+      something.find(item => g(item, 'abc') === 'def')`,
+      errors: [{ messageId: 'main', type: 'CallExpression' }],
+    },
+    {
+      // findBy with two arguments and `get` is imported from package other than @ember/object
+      code: `import { get as g } from 'dummy';
+      something.findBy('abc', 'def')`,
+      output: `import { get } from '@ember/object';
+import { get as g } from 'dummy';
+      something.find(item => get(item, 'abc') === 'def')`,
+      errors: [{ messageId: 'main', type: 'CallExpression' }],
+    },
```

Comment:
Let's make sure we have valid test cases for `findBy` with the single arg and double arg versions.

Reply:
@bmish We already have one test case covering single arg version
https://github.com/ember-cli/eslint-plugin-ember/pull/1614/files#diff-96ccefab5cb5eeb84d694bfe79faa534ff699ef8cf543605c4711812c47a896dR423

Reply:
Will merge this once we address this comment.

Reply:
Ah, got it. Just added the valid tests.